go.mod

```diff
@@ -174,7 +175,6 @@ require (
 	golang.org/x/exp v0.0.0-20231226003508-02704c960a9b // indirect
 	golang.org/x/mod v0.17.0 // indirect
 	golang.org/x/oauth2 v0.18.0 // indirect
-	golang.org/x/sync v0.7.0 // indirect
```

Comment:
it seems that the go.mod file has not changed substantially. you need to restore it.

Reply:
difference in indirect

Reply:
I don't think it's necessary.